src/LiveComponent/assets/test/diff-dom.test.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    htmlToElement,
+} from '../src/dom_utils';
+import { DiffDOM } from "diff-dom"
+
+describe('diff-dom', () => {
+    it('Just playing', () => {
+
+
+        const dd = new DiffDOM();
+
+        const from = htmlToElement(`
+        <div>
+            Text at the top
+            <div class="child-element">Child1 text</div>
+            <div>between</div>
+            <div class="child-element">Child2 text<span class="foo">icon</span></div>
+        </div>
+        `)
+
+        const to = htmlToElement(`
+        <div>
+            Text at the top
+            <div class="child-element">Child1 text</div>
+            <div>between</div>
+            <div class="child-element">Child2 text</div>
+        </div>
+        `)
+
+        const diff = dd.diff(from, to)
+        console.log(diff);
```

Comment:
Shouldn't be an assertion insteadn?

Reply:
this was totally committed by mistake (by me). Removed now. It's something I'm playing with for live components...